rocksdb/tests/test_options.py

```diff
@@ -165,7 +202,88 @@ def test_compaction_opts_universal(self):
         self.assertEqual(2, uopts['min_merge_width'])
         self.assertEqual(30, uopts['max_merge_width'])
 
-    def test_row_cache(self):
+    def test_rockdb_options(self):
+        NOTNONE = object()
+        UNSETTABLE = object()
+        for option, def_value, new_value in (
+                ('max_open_files', NOTNONE, 10),
+                ('row_cache', None, UNSETTABLE),
+                ('max_file_opening_threads', NOTNONE, 10),
+                ('max_total_wal_size', NOTNONE, 10),
+                ('max_background_jobs', NOTNONE, 10),
+                ('base_background_compactions', NOTNONE, 10),
+                ('max_background_compactions', NOTNONE, 10),
+                ('max_subcompactions', NOTNONE, 10),
+                ('max_background_flushes', NOTNONE, 10),
+                ('max_log_file_size', NOTNONE, 10),
+                ('log_file_time_to_roll', NOTNONE, 10),
+                ('recycle_log_file_num', NOTNONE, 10),
+                ('stats_history_buffer_size', NOTNONE, 10),
+                ('max_manifest_file_size', NOTNONE, 10),
+                ('table_cache_numshardbits', NOTNONE, 10),
+                ('wal_ttl_seconds', NOTNONE, 10),
+                ('wal_size_limit_mb', NOTNONE, 10),
+                ('manifest_preallocation_size', NOTNONE, 10),
+                ('allow_mmap_reads', False, True),
+                ('allow_mmap_writes', False, True),
+                ('use_direct_reads', False, True),
+                ('use_direct_io_for_flush_and_compaction', False, True),
+                ('allow_fallocate', True, False),
+                ('is_fd_close_on_exec', True, False),
+                ('skip_log_error_on_recovery', False, True),
+                ('stats_dump_period_sec', 600, 3600),
+                ('stats_persist_period_sec', 600, 3600),
+                ('persist_stats_to_disk', False, True),
+                ('stats_history_buffer_size', 1024*1024, 1024),
```

Comment:
you have 2 tests for this option, is that on purpose?

Reply:
That was a mistake. Removed it.